qdrant-landing/content/documentation/fastembed/fastembed-colbert.md

```diff
@@ -5,33 +5,58 @@ weight: 6
 
 # How to Generate ColBERT Multivectors with FastEmbed
 
-With FastEmbed, you can use ColBERT to generate multivector embeddings. ColBERT is a powerful retrieval model that combines the strength of BERT embeddings with efficient late interaction techniques. FastEmbed will provide you with an optimized pipeline to utilize these embeddings in your search tasks.
+Qdrant supports [multivector representations](https://qdrant.tech/documentation/concepts/vectors/#multivectors). 
+With FastEmbed, you can use ColBERT to generate multivector embeddings; FastEmbed will provide an optimized pipeline to utilize these embeddings in your search tasks.
 
-Please note that ColBERT requires more resources than other no-interaction models. We recommend you use ColBERT as a re-ranker instead of a first-stage retriever. 
+ColBERT produces one vector per token (a `token` is a meaningful text unit for a machine learning model). Consequently,
+ColBERT is more precise than dense embedding models like `BAAI/bge-small-en-v1.5`, which embed a whole text into just a single vector. 
```

Comment:
I think this sentence should be extended to highlight the idea: ColBERT produces one vector per token, which allows it to preserve more contextual information comparing to ...
Otherwise, it might be perceived as:
ColBERT produces one vector per token => ColBERT is more precise

Reply:
imho, claims like /is more precise/, /requires significantly more resources/, etc., are too confident, and assumes that it is true in all the cases, however, it might not be true. Instead, I would suggest using something like it might be more precise in some cases, it in general requires more resources, etc.

Reply:
Might be worth mentioning, that colbert might be a powerful retriever. 
Otherwise, we are speaking about colbert as a good reranker, but provide an example of it being first-stage retriever.